lib/logstash/filters/elasticsearch.rb

```diff
@@ -135,11 +122,15 @@ class LogStash::Filters::Elasticsearch < LogStash::Filters::Base
   # What status codes to retry on?
   config :retry_on_status, :validate => :number, :list => true, :default => [500, 502, 503, 504]
 
+
+  config :ssl, :obsolete => "Set 'ssl_enabled' instead."
+  config :ca_file, :obsolete => "Set 'ssl_certificate_authorities' instead."
+  config :keystore, :obsolete => "Set 'ssl_keystore_path' instead."
+  config :keystore_password, :obsolete => "Set 'ssl_keystore_password' instead."
```

Comment:
```suggestion
  config :keystore_password, :validate => :password, :obsolete => "Set 'ssl_keystore_password' instead."
```
Keep validation for obfuscation purposes.

Reply:
Nice catch!